### PR TITLE
Remove k2v/v2k and document multisecret

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: lsstvaultutils/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
+from typing import Sequence
 from .tokenadmin import AdminTool
 from .recursivedeleter import RecursiveDeleter
-from .secretcopier import SecretCopier
 from .vaultconfig import VaultConfig
 
-__all__ = [AdminTool, RecursiveDeleter, SecretCopier, VaultConfig]
+__all__: Sequence[str] = ["AdminTool", "RecursiveDeleter", "VaultConfig"]
```
Comment (rra) [inline lsstvaultutils/__init__.py]: Nit: You don't need a type here. mypy will figure out that it's a list of str.

File: lsstvaultutils/recursivedeleter.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 """
 
 import click
-import hvac
+import hvac  # type: ignore
```
Comment (rra) [inline lsstvaultutils/recursivedeleter.py:6]: Just curious, what error was this producing without the comment?

**Review by rra — APPROVED** [review]
Seems fine although there's a bug somewhere if all those ignore comments are needed.

File: lsstvaultutils/recursivedeleter.py (hunk 2 above)
Comment (athornton) [reply lsstvaultutils/recursivedeleter.py:6]: When you lint it with mypy it warns that hvac has no type hints.  

```
lsstvaultutils/recursivedeleter.py:6: error: Skipping analyzing 'hvac': found module but no type hints or library stubs
```

File: lsstvaultutils/__init__.py (hunk 1 above)
Comment (athornton) [reply lsstvaultutils/__init__.py]: Thanks.

**Comment by rra** [comment]
Oh, mypy configuration issue. Generally I use the following in `setup.cfg`:

```
[mypy]
disallow_untyped_defs = True
disallow_incomplete_defs = True
ignore_missing_imports = True
show_error_codes = True
strict_equality = True
warn_redundant_casts = True
warn_unreachable = True
warn_unused_ignores = True
```

For this problem, `ignore_missing_imports` is the important part. A lot of libraries aren't typed yet and you don't want to have to put `# type: ignore` markers on all of them.

**Pull request merged**